### PR TITLE
Bubble-sort Project-1 Ruby

### PR DESCRIPTION
**Pull request by garciajordy**
Bubble_sort
-Create a method that sorts arrays by checking if the next value is greater or not and changing their index.

Bubble_sort_by
-Create a method that sorts arrays by checking if the next text length is greater or not and changing their index.
-Using yield to change bubble_sort to bubble_sort_by

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @JulianaOsemeke and @garciajordy  :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this milestone can be approved.

## REQUIRED

* [ ] Your bubble_sort method is unable to sort an array of numbers in increasing order.
* [ ] You have a bubble_sort_by file but you don't have a bubble_sort_by method


### Make the required changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

File: bubblesort.rb
Comment (BrianSammit) [inline bubblesort.rb]: In line 18 you have this code `p bubble_sort([4, 3, 78, 2, 0, 2])` this code is debugging code. You use it to test if the code works but that's debugging code. 
Please delete it or if you want, keep it in another branch.

File: bubble_sort_by.rb
Comment (BrianSammit) [inline bubble_sort_by.rb]: In line 18 you have this code `bubble_sort_by(%w[hi hello hey hhee holapol]) { |left, right| left.length - right.length }` this code is debugging code. You use it to test if the code works but that's debugging code. 
Please delete it or if you want, keep it in another branch.

**Review by BrianSammit — COMMENTED** [review]
CHANGES REQUIRED :recycle: :red_circle: 
-------------
Hello @garciajordy @JulianaOsemeke You did an amazing job :clap: I like how you created the logic it's really clear :+1: 
However, there are a few changes to make :hammer_and_wrench: see them below :eyes: :point_down: 
--------------

- [ ] The gif image in the Readme it's not loading please fix it. You are calling the image but you actually don't have the image in the directory. 
- [ ] You have a debugging code. Remember you must not commit debugging code. [Here](https://questions.microverse.org/t/debugging-instructions-code/346) you can read about why. See my inline comment to know what are the debugging code. :eyes: 
-------------------
Make these minor changes and send it for another review. :wrench: :muscle: 
HAPPY CODING!!

**Review by BrianSammit — APPROVED** [review]
STATUS: APPROVED 🟢 💪 
----------
Hello, again @garciajordy @JulianaOsemeke Great job working on the required changes. 👏 
🥳 

- [ ] For the next projects keep in mind the formatting rules for your commit messages. You must write your commit messages in the imperative. [Here](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules) you can read about the formatting rules for commits messages. [Here](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message) you can read about how to change a commit message. 
----------
Now you can merge this branch 👏 💯 
If you have any questions reach me on SLACK
@ Brian Sammit Cruz Rodriguez 
HAPPY CODING!! AND GOOD LUCK WITH THE COMING PROJECTS 🍀 🤞 

Well-Done-Applause

**Pull request merged**
